core/setup.py

```diff
@@ -75,8 +75,8 @@
         "minimal-snowplow-tracker>=0.0.2,<0.1",
         "dbt-semantic-interfaces>=0.5.1,<0.6",
         # Minor versions for these are expected to be backwards-compatible
-        "dbt-common>=1.0.2,<2.0",
-        "dbt-adapters>=0.1.0a2,<2.0",
+        "dbt-common>=1.0.4,<2.0",
```

Comment:
Do we want to set bounds to patch versions, or is minor version sufficient?

For external packages, it looks like we default to pinning to minor version (unless there’s a bug that got fixed in a patch version).

Reply:
@aranke `dbt-core==1.8.0` is actually incompatible with `dbt-common==1.0.3` and `dbt-adapters==1.1.0`, due to the `__post_serialize__` signature change. I think we need to put the specific patch numbers as the minima here to make clear that incompatibility. It wasn't our proudest semver moment, but we're learning from it and moving forward :)